scripts/apps/workspace/content/services/ContentService.ts

```diff
@@ -135,24 +135,6 @@ export function ContentService(api, templates, desks, packages, archiveService,
         });
     };
 
-    /**
-     * Create new item using given content type
-     *
-     * @param {Object} contentType
-     * @return {Promise}
-     */
-    this.createItemFromContentType = function(contentType) {
```

Comment:
@petrjasek I think we don't need this anymore. I couldn't find it's being used anywhere else now. This was the [PR](https://github.com/superdesk/superdesk-client-core/commit/29dd7b3b97d9d08c14ecd3ddbb8ec36e81685902) where we removed its use.